docs/Release_1.0.8.md

```diff
@@ -0,0 +1,85 @@
+## [Horizen](https://horizen.io/) Desktop GUI Wallet binary release 1.0.8
+
+It includes [Horizen 3.2.1 binaries](https://github.com/HorizenOfficial/zen/releases/tag/v3.2.1). 
+
+**This wallet is targeted at advanced users who understand the implications of running a full Zen node on**
+**the local machine, maintaining a full local copy of the blockchain, maintaining and backing up the**
+**Zen nodes's `wallet.dat` file etc! The wallet is not suitable for novice crypto-currency users!**
+
+**SECURITY WARNING: Encryption of the wallet.dat file is not yet supported for Horizen. Using the wallet** 
+**on a system infected with malware may result in wallet data/funds being stolen. The**
+**wallet.dat needs to be backed up regularly (not just once - e.g. after every 30-40**
+**outgoing transactions) and it must also be backed up after creating a new Z address.**
+
+**STABILITY WARNING: The GUI wallet is as yet considered experimental! It is known to exhibit occasional stability problems related to running a full Zen node.**
+**Specifically if the locally running `zend` cannot start properly due to issues with the local blockchain, the GUI cannot start either!**
+**Users need to be prepared to fix such problems manually as described in the [troubleshooting guide](https://github.com/HorizenOfficial/zencash-swing-wallet-ui/blob/master/docs/TroubleshootingGuide.md).**
+**Doing so requires command line skills.**
+
+**AUTO-DEPRECATION WARNING: Wallet binary releases for Mac/Windows contain ZEN full node binaries. These have an auto-deprecation feature:**
+**they are considered outdated after 16 weeks and stop working. So they need to be updated to a newer version before this term expires.**
+**Users need to ensure they use an up-to-date version of the wallet (e.g. update the wallet every two months or so).**
+
+### Installing the Horizen Desktop GUI Wallet on Windows
+
+It requires a 64-bit Windows 7 or later version to run.
+
+1. Download the Wallet EXE file
+[HorizenDesktopGUIWallet-1.0.8.exe](https://github.com/HorizenOfficial/zencash-swing-wallet-ui/releases/download/1.0.8/HorizenDesktopGUIWallet-1.0.8.exe).
+
+2. Security check: You may decide to run a virus scan on it, before proceeding... In addition using a tool 
+such as [http://quickhash-gui.org/](http://quickhash-gui.org/) you may calculate the its SHA256 checksum. The 
+result should be:
+```
+dfcd5bbef381759c4634d94eec93968d8843b80ccd1b6583495facb11b3ee5cf  HorizenDesktopGUIWallet-1.0.8.exe
```

Comment:
To be updated after merging this PR to development, then tagging the latest commit on development as `1.0.8` and getting the sha256 hash from *.sha256 release artifact and commiting the changes on `development` branch.

Once done open a PR from `development` to `master`.